src/tox/config/sets.py

```diff
@@ -199,7 +199,7 @@ def work_dir_builder(conf: Config, env_name: str | None) -> Path:  # noqa: U100
         self.add_config(
             keys=["temp_dir"],
             of_type=Path,
-            default=lambda conf, _: cast(Path, self["tox_root"]) / ".tmp",  # noqa: U100, U101
+            default=lambda conf, _: cast(Path, self["tox_root"]) / ".temp",  # noqa: U100, U101
```

Comment:
Why can you change code without breaking any tests? Is there a test missing?

Reply:
Primarily because this is an open-source project, and there's only so much time I can spend on this project. I've only got constrained to helping out on it.